Assets/MixedRealityToolkit-SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -122,21 +128,26 @@ public bool UpdateSolvers
 
         protected readonly List<Solver> solvers = new List<Solver>();
 
+        /// <inheritdoc />
+        public override bool IsFocusRequired => false;
+
         private float lastUpdateTime;
         private GameObject transformWithOffset;
 
         #region MonoBehaviour Implementation
 
-        private void Awake()
+        private async void Awake()
```

Comment:
Doesn't making Awake asychronous violate Unity's order of operations contract?
https://docs.unity3d.com/Manual/ExecutionOrder.html

Though the Awake() docs don't specifically say Awake cannot be async, it does say it can't be a coroutine:
https://docs.unity3d.com/ScriptReference/MonoBehaviour.Awake.html

It seems safer to stick with the spirit of Awake() and finish up before any Start()s get called.

Reply:
It does't change the timing of the order of operations.

Reply:
But if it is awaited, then yes, it'll delay that call.